### ISSUE TITLE
Can't pay for pools on devnet

### ISSUE THREAD

**Issue by waleedhammam** [threefoldtech/tfexplorer#336]
Try to create a pool on devnet, pay for it. 

It's not marked as paid even

![Screenshot from 2021-06-27 13-30-02](https://user-images.githubusercontent.com/10658229/123542985-cfe04580-d74c-11eb-9579-9bb5eae87dcd.png)

**Comment by xmonader** [comment]
logs:
```
un 27 13:39:18 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:18+02:00 debug 0 queue
Jun 27 13:39:18 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:18 +0200] "GET /api/v1/reservations/nodes/9PdutHsdDSxcKUUyDg8ovS1KWh47qLT5R9h5uoFgRUH2/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug 0 queue
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:20 +0200] "GET /api/v1/reservations/nodes/Gr8NxBLHe7yjSsnSTgTqGr7BHbyAUVPJqs8fnudEE4Sf/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 info fetching balance for address address=GC4NEE5M7X6S3SEHTEV46HOHDV7Z6W3JKKKFSBZRCAZUJASCRQ6G5YRL
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 error operation failed error="failed to get balance: could not get transactions: horizon error: \"Resource Missing\" - check horizon.Error.Problem for more information" memo=p-12264 operation=Refund problem=map[] status="404 Not Found" status-code=404
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55535
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55536
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55537
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55538
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55539
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55540
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55541
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55542
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55543
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55544
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55545
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55550
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55551
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55552
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55553
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55554
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55555
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55556
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55557
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55558
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55559
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55561
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55562
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug let's deploy func=pipeline.Next id=55563
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:20 +0200] "GET /api/v1/reservations/workloads?customer_tid=3874&next_action=3&page=1 HTTP/1.1" 200 81608
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:20+02:00 debug 0 queue
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:20 +0200] "GET /api/v1/reservations/nodes/qzuTJJVd5boi6Uyoco1WWnSgzTb7q8uN79AjBT9x9N3/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:20 +0200] "GET /api/v1/gateways?page=1 HTTP/1.1" 200 3410
Jun 27 13:39:20 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:20 +0200] "GET /api/v1/reservations/pools/owner/3874?page=1 HTTP/1.1" 200 1239
Jun 27 13:39:21 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:21+02:00 info fetching balance for address address=GC4NEE5M7X6S3SEHTEV46HOHDV7Z6W3JKKKFSBZRCAZUJASCRQ6G5YRL
Jun 27 13:39:21 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:21+02:00 debug 0 queue
Jun 27 13:39:21 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:21 +0200] "GET /api/v1/reservations/nodes/3NAkUYqm5iPRmNAnmLfjwdreqDssvsebj4uPUt9BxFPm/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:21 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:21+02:00 error operation failed error="failed to get balance: could not get transactions: horizon error: \"Resource Missing\" - check horizon.Error.Problem for more information" memo=p-12264 operation=Refund problem=map[] status="404 Not Found" status-code=404
Jun 27 13:39:22 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:22+02:00 debug 0 queue
Jun 27 13:39:22 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:22 +0200] "GET /api/v1/reservations/nodes/48YmkyvfoXnXqEojpJieMfFPnCv2enEuqEJcmhvkcdAk/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:23 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:23+02:00 debug 0 queue
Jun 27 13:39:23 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:23 +0200] "GET /api/v1/reservations/nodes/BBmMHCw392RLiByVEUT2ac6y6qJpbW9xj8TRdgk5rMGN/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:24 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:24+02:00 info fetching balance for address address=GC4NEE5M7X6S3SEHTEV46HOHDV7Z6W3JKKKFSBZRCAZUJASCRQ6G5YRL
Jun 27 13:39:24 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:24+02:00 error operation failed error="failed to get balance: could not get transactions: horizon error: \"Resource Missing\" - check horizon.Error.Problem for more information" memo=p-12264 operation=Refund problem=map[] status="404 Not Found" status-code=404
Jun 27 13:39:26 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:26+02:00 info fetching balance for address address=GC4NEE5M7X6S3SEHTEV46HOHDV7Z6W3JKKKFSBZRCAZUJASCRQ6G5YRL
Jun 27 13:39:26 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:26+02:00 error operation failed error="failed to get balance: could not get transactions: horizon error: \"Resource Missing\" - check horizon.Error.Problem for more information" memo=p-12264 operation=Refund problem=map[] status="404 Not Found" status-code=404
Jun 27 13:39:26 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:26+02:00 debug 0 queue
Jun 27 13:39:26 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:26 +0200] "GET /api/v1/reservations/nodes/BpTAry1Na2s1J8RAHNyDsbvaBSM3FjR4gMXEKga3UPbs/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:27 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:27+02:00 debug 0 queue
Jun 27 13:39:27 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:27 +0200] "GET /api/v1/reservations/nodes/2anfZwrzskXiUHPLTqH1veJAia8G6rW2eFLy5YFMa1MP/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:27 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:27+02:00 debug 0 queue
Jun 27 13:39:27 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:27 +0200] "GET /api/v1/reservations/nodes/FTothsg9ZuJubAEzZByEgQQUmkWM637x93YH1QSJM242/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:27 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:27+02:00 info fetching balance for address address=GC4NEE5M7X6S3SEHTEV46HOHDV7Z6W3JKKKFSBZRCAZUJASCRQ6G5YRL
Jun 27 13:39:27 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:27+02:00 error operation failed error="failed to get balance: could not get transactions: horizon error: \"Resource Missing\" - check horizon.Error.Problem for more information" memo=p-12264 operation=Refund problem=map[] status="404 Not Found" status-code=404
Jun 27 13:39:27 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:27+02:00 debug 0 queue
Jun 27 13:39:27 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:27 +0200] "GET /api/v1/reservations/nodes/DJHD5cMjRkuVh4enQ6dqCAuPNUPfH2a45gLv7JDDswbT/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:28 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:28+02:00 debug 0 queue
Jun 27 13:39:28 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:28 +0200] "GET /api/v1/reservations/nodes/9PdutHsdDSxcKUUyDg8ovS1KWh47qLT5R9h5uoFgRUH2/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:30 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:30+02:00 debug 0 queue
Jun 27 13:39:30 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:30 +0200] "GET /api/v1/reservations/nodes/Gr8NxBLHe7yjSsnSTgTqGr7BHbyAUVPJqs8fnudEE4Sf/workloads?from=56580 HTTP/1.1" 200 5
Jun 27 13:39:30 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:30+02:00 info fetching balance for address address=GC4NEE5M7X6S3SEHTEV46HOHDV7Z6W3JKKKFSBZRCAZUJASCRQ6G5YRL
Jun 27 13:39:30 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:30+02:00 error operation failed error="failed to get balance: could not get transactions: horizon error: \"Resource Missing\" - check horizon.Error.Problem for more information" memo=p-12264 operation=Refund problem=map[] status="404 Not Found" status-code=404
Jun 27 13:39:30 v2hq.dev.grid.tf tfexplorer[721927]: 2021-06-27T13:39:30+02:00 debug 0 queue
Jun 27 13:39:30 v2hq.dev.grid.tf tfexplorer[721927]: 127.0.0.1 - - [27/Jun/2021:13:39:30 +0200] "GET /api/v1/reservations/nodes/qzuTJJVd5boi6Uyoco1WWnSgzTb7q8uN79AjBT9x9N3/workloads?from=56580 HTTP/1.1" 200 5

```

**Comment by LeeSmet** [comment]
`check horizon.Error.Problem for more information` -> devnet is running against our own horizon server (to not be rate limited on the payment aggregation FR). This error would indicate that the historical state is not ingested (horizon only ingests new state, ingesting old state needs to be triggered manually). The fix would be to move back to the official horizon for now, and remove the aggregation of payments feature again for the time being. Though I'm not really a fan of this and its only devnet.

Tl;dr: blocked on github.com/threefoldtech/tf_operations/issues/157 and github.com/threefoldtech/tf_operations/issues/185
